src/app/index.tsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './app';
-// import '@babel/polyfill';
+import App from './app.tsx';
```

Comment:
Is there another way to resolve .tsx extensions in the web pack config / Eslint rules?

Reply:
there is a way to achieve this in webpack.config.js but I'm not sure I know how atm.